project.clj

```diff
@@ -30,14 +31,14 @@
                     :source-paths ["src" "env/dev"]
```

Comment:
Is this still needed?

Reply:
well, everything still works, when I remove "src" part, but not "env/dev". I will try to investigate into this.

Reply:
looks like these are in place now.